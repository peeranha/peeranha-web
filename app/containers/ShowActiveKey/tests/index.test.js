import { ShowActiveKey } from '../index';

const cmp = new ShowActiveKey();
cmp.props = {
  showActiveKeyDispatch: jest.fn(),
  hideActiveKeyModalDispatch: jest.fn(),
  showActiveKeyModalDispatch: jest.fn(),
  children: null,
  showActiveKeyProcessing: false,
  showModal: true,
  locale: 'en',
};

describe('<ShowActiveKey />', () => {
  it('render', () => {
    expect(cmp.render()).toMatchSnapshot();
  });
});
