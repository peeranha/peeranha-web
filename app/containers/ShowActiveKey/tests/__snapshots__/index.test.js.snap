// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShowActiveKey /> render 1`] = `
<React.Fragment>
  <UNDEFINED
    closeModal={[MockFunction]}
    show={true}
  >
    <ReduxForm
      locale="en"
      showActiveKey={[MockFunction]}
      showActiveKeyProcessing={false}
    />
  </UNDEFINED>
  <button
    onClick={[MockFunction]}
  />
</React.Fragment>
`;
